rust/core-lib/src/plugins/manifest.rs

```diff
@@ -96,6 +96,19 @@ pub enum PluginActivation {
 }
 
 #[cfg(not(target_os = "fuchsia"))]
```

Comment:
My mistake, sloppy rebasing in the Github web UI. I just fixed it and did it as an actual rebase without a merge commit.